README.md

```diff
@@ -21,3 +21,11 @@ Tips for using GPT for Confluence:
 - Keep your requests as specific as possible to get the most accurate results.
 - Don't rely solely on generated text - always review and edit to ensure it meets your needs.
 - Experiment with different types of content and formats to fully take advantage of the tool's flexibility.
+
+Develop:
+-   VSCode + node v16.13.2
+-   configure **ngrok** in VSCode. reference: https://ngrok.com/blog-post/integrating-vscode-and-ngrok
```

Comment:
VSCode (recommended)
node v16.20.0
Set up **Ngrok**: use [VSCode extendsion](https://ngrok.com/blog-post/integrating-vscode-and-ngrok) or ngrok command to setup your own API key

Reply:
mybe rewirte to : "node v16.*.*"?  because I was using  node v16.20.0